sechub-storage-s3-aws/src/main/java/com/mercedesbenz/sechub/storage/s3/AwsS3TransferManagerFactory.java

```diff
@@ -0,0 +1,12 @@
+package com.mercedesbenz.sechub.storage.s3;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.transfer.TransferManager;
+import com.amazonaws.services.s3.transfer.TransferManagerBuilder;
+
+public class AwsS3TransferManagerFactory {
+    public TransferManager create(AmazonS3 client) {
+        long multipartUploadThreshold = 5 * 1024 * 1025; // threshold 5 MiB
```

Comment:
I think there is a typo in this line.
```suggestion
        long multipartUploadThreshold = 5 * 1024 * 1024; // threshold 5 MiB
```